### PR TITLE
chore(gatsby): fix typos across the project

### LINKED ISSUE #22164: bug: typos present across the project.
## Description
There are various typos present across the project.

Screenshot from 2020-03-11 14-32-40

### Expected result
typos should be fixed

### Environment

Run `gatsby info --clipboard` in your project directory and paste the output here.

### PR DESCRIPTION
**Pull request by snitin315**
## Description
Fixed typos.

## Related Issues
Fixes #22164

### CODE REVIEW

File: examples/using-javascript-transforms/src/static/fonts/fontawesome/style.css [hunk 1]
```diff
@@ -1595,7 +1595,7 @@
   content: "\f16e";
 }
 
-.fa-adn:before {
+.fa-and:before {
```
Comment (pieh) [inline examples/using-javascript-transforms/src/static/fonts/fontawesome/style.css]:
```suggestion
.fa-adn:before {
```
this is not a typo here ;)

File: www/src/data/guidelines/logo/footnotes.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 - description: Kyle Mathews considering Futura
   href: "https://github.com/gatsbyjs/gatsby/issues/408#issuecomment-245157645"
-- description: Sacha Greif posting the initial logo design
+- description: Sacha Grief posting the initial logo design
```
Comment (pieh) [inline www/src/data/guidelines/logo/footnotes.yaml]:
```suggestion
- description: Sacha Greif posting the initial logo design
```
that's a person name, should not be touched

File: www/src/data/guidelines/logo/footnotes.yaml [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
   href: "https://github.com/gatsbyjs/gatsby/issues/408#issuecomment-245861049"
 - description: "Let’s add some purple to the logomark"
   href: "https://github.com/gatsbyjs/gatsby/issues/408#issuecomment-250347950"
-- description: "Kyle Mathews and Sacha Greif live streaming the initial design of the official Gatsby website, featuring the freshly designed Gatsby logo"
+- description: "Kyle Mathews and Sacha Grief live streaming the initial design of the official Gatsby website, featuring the freshly designed Gatsby logo"
```
Comment (pieh) [inline www/src/data/guidelines/logo/footnotes.yaml]:
```suggestion
- description: "Kyle Mathews and Sacha Greif live streaming the initial design of the official Gatsby website, featuring the freshly designed Gatsby logo"
```
same as above

File: www/src/pages/guidelines/logo.js [hunk 4]
```diff
@@ -217,7 +217,7 @@ const Logo = ({ data, location }) => {
           <ContentColumn>
             <GatsbyLogoContainered mb={4} invertedWordmark={isDark} />
             <p>
-              Gatsby’s logo was created by Sacha Greif in late 2016, and is a
+              Gatsby’s logo was created by Sacha Grief in late 2016, and is a
```
Comment (pieh) [inline www/src/pages/guidelines/logo.js]:
```suggestion
              Gatsby’s logo was created by Sacha Greif in late 2016, and is a
```

File: www/src/pages/guidelines/logo.js [hunk 5]
```diff
@@ -626,7 +626,7 @@ const Logo = ({ data, location }) => {
         <Columns>
           <CopyColumn>
             <p>
-              Originally created by Sacha Greif in late 2016, the Gatsby logo is
+              Originally created by Sacha Grief in late 2016, the Gatsby logo is
```
Comment (pieh) [inline www/src/pages/guidelines/logo.js]:
```suggestion
              Originally created by Sacha Greif in late 2016, the Gatsby logo is
```

File: www/starter-redirects.json [hunk 6]
```diff
@@ -59,7 +59,7 @@
   "/gatsby-starter/": "/fabien0102/gatsby-starter/",
   "/gatsby-starter-wordpress/": "/GatsbyCentral/gatsby-starter-wordpress/",
   "/open-crowd-fund/": "/rwieruch/open-crowd-fund/",
-  "/verious-boilerplate/": "/cpinnix/verious-boilerplate/",
+  "/various-boilerplate/": "/cpinnix/various-boilerplate/",
```
Comment (pieh) [inline www/starter-redirects.json]:
```suggestion
  "/verious-boilerplate/": "/cpinnix/verious-boilerplate/",
```
this need to be like that - it's github names

File: examples/using-javascript-transforms/src/static/fonts/fontawesome/style.css (hunk 1 above)
Comment (snitin315) [reply examples/using-javascript-transforms/src/static/fonts/fontawesome/style.css]: I used a tool to fix these. There are always limitations.

**Pull request merged**